examples/src/complex/ComplexExample.js

```diff
@@ -5,13 +5,12 @@ import {
 } from '@sandstreamdev/react-swipeable-list';
 import '@sandstreamdev/react-swipeable-list/dist/styles.css';
 
-import ListItem from './ListItem';
-import ItemContent from './ItemContent';
+import ComplexListItem from '../list-items/ComplexListItem';
+import ComplexSwipeContent from '../list-items/ComplexSwipeContent';
 import { MailIcon, ReplyIcon, DeleteIcon } from '../../images/icons';
+import './ComplexExample.css';
```

Comment:
Consider spacing out the side-effectful imports.

```suggestion

import './ComplexExample.css';
```